Flaky test `TestNatStun`

```shell
--- FAIL: TestNatStun (10.82s)
    stun_test.go:29: 
        	Error Trace:	C:/projects/go-ethereum/p2p/nat/stun_test.go:29
        	Error:      	Received unexpected error:
        	            	STUN requests failed
        	Test:       	TestNatStun
```

https://ci.appveyor.com/project/ethereum/go-ethereum/builds/51545247/job/54xj91ka37dgylkt

--- a/p2p/nat/stun_test.go
+++ b/p2p/nat/stun_test.go
@@ -18,16 +18,7 @@ package nat
 
 import (
 	"testing"
-
-	"github.com/stretchr/testify/assert"
 )
-
-func TestNatStun(t *testing.T) {
-	nat, err := newSTUN("")
-	assert.NoError(t, err)
-	_, err = nat.ExternalIP()
-	assert.NoError(t, err)
-}
 
 func TestUnreachedNatServer(t *testing.T) {
 	stun := &stun{

